wiki/Beatmap/Overall_difficulty/en.md

```diff
@@ -30,6 +30,8 @@ Hit windows for judgments can be viewed when hovering over the [beatmap informat
 | 100 | `140 - 8 * OD` |
 | 50 | `200 - 10 * OD` |
 
+![](/wiki/shared/ODTable.png "Comparison of hit windows for different combinations of OD and game modifiers. For Half Time and Double Time combinations, the referenced OD values are only valid for the hit windows of 300s, and would be different for 100s and 50s.")
```

Comment:
I think saying "referenced OD values" is wrong because those are the final result and not referenced by anything on the chart. maybe somethin like "the OD values are based on..." or "the OD values are calculated using..."

Reply:
this is not the actual OD value. instead, it's a value used as an example, the hit window of which for a 300 is the same as the hit window for a given OD and a combination of mods. doesn't that make it a value used as a reference? perhaps I should have used `reference` instead of `referenced`? 

if I am still wrong, which I admit I may be, well, fine, I'll make an edit (or you can too, whichever is faster and more comfortable)

Reply:
no the word itself just doesn't make sense at all here. there's nothing that we're referencing from.

```suggestion
![](/wiki/shared/ODTable.png "Comparison of hit windows for different combinations of OD and game modifiers. For Half Time and Double Time combinations, the shown OD values are only valid for the hit windows of 300s, and would be different for 100s and 50s.")
```

Reply:
okay, done